packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/index.tsx

```diff
@@ -26,6 +27,15 @@ import { TNil, ReduxState } from '../../../types';
 import { Span, Trace } from '../../../types/trace';
 
 import './index.css';
+import { createStyle, Theme, withTheme } from '../Theme';
+
+const getStyles = createStyle(theme => {
+  return {
+    TraceTimelineViewer: css`
+      border-bottom: ${theme.borderStyle};
```

Comment:
Is the intent to provide all styles through `context`, or only configurable styles?